src/guides/v2.4/security/two-factor-authentication.md

```diff
@@ -18,7 +18,6 @@ Two-Factor Authentication gives you the ability to:
 Magento has new workflows for _Admin_ users, including:
 
 -  The ability to configure the 2FA provider globally or individually.
--  _Admin_ users set their own personal 2FA at first login, and receive a confirmation email to verify their identity.
```

Comment:
This particular line doesn't say anything about a provider.
The one above it does. As an admin, you do set 2FA at first login and receive a confirmation email to verify identity, don't you? Do you mean that they do not configure provider "individually" ?

Reply:
@hguthrie From my own perspective and others that I've discussed this part of the docs with that have gotten confused by this line, It is saying that admins can set their own personal 2FA **(provider)** if they want to but that is only the case on the initial configuration of 2FA. You cannot and have not been able to set your own provider after that initial global configuration.

Do you believe it is better to keep this line and reword it instead?

Reply:
@MarkBerube Not necessarily. I needed a little more information about it to understand where the confusion is and you said it perfectly! Thanks! Based on your reply, it does seem the current content flow is confusing. I think it may need a little more fix than just removing one line. For instance, maybe instead of having this generic list of workflows, perhaps there needs to be something more helpful that distinguishes the time qualifier you mentioned. "...only the case on the initial configuration..." sounds **important** to know to me. 🙂

Reply:
So, I am looking into this a little deeper. The User Guide talks about setting up provider as well: https://docs.magento.com/user-guide/stores/security-two-factor-authentication.html
Those updates were made in a January PR: https://github.com/magento/merchdocs/pull/970

I'll continue to look into this. It's better to make sure all mention of adding or updating provider is clear and consistent across User and Devdocs.